tests/test-webapps/test-simple-session-webapp/src/main/webapp/index.jsp

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
Probably don't need this file: you only ever hit the `/session` url, don't you? If it is really necessary to have it, lets make it a static index.html, rather than bringing `jsp` into the mix.